docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+```shell
+git push heroku <branch_name>:master
+```
+Now that our Frontend code is uploaded to Heroku, we can use the url Heroku gave us to open our application online. Now, anyone with this link will be able to see our application.
+To open our application we can run:
+```shell 
+heroku open
+```
+This should open a browser tab with our project's home page. 
+However, this is only the user interface portion of our application. Now we have to deploy our backend code.
+
+
+### Deploy backend
+To deploy our backend application, first we have to configure our application. First, we have to navigate to our pom.xml 
+file which gets created after every maven project. We have to configure all our app's dependencies through maven so that when we export our Java project our dependencies also get exported.
+An app's dependencies is external code that other people wrote that we use in our application. Usually, if we did not have problems during local development, then our pom.xml file should be fine and ready to be exported.
+If not, then now is the time to fix it. Potential fixes include using different versions of a dependency or getting rid of dependencies that we did not end up using.
+Next, we have to create a Procfile. This is a text-like file that that acts almost like a configuration file that Heroku uses to set up your java application. Our procfile must look like:
+
+```ignorelang
+web: java $JAVA_OPTS -cp target/classes:target/dependency/* <project main package>
+```
+
+In our case, if you download our catch-up code, our procfile looks like:
+```ignorelang
+web: java $JAVA_OPTS -jar service/target/service-1.0-SNAPSHOT-jar-with-dependencies.jar
+```
+
+What this procfile is doing is specifying that our application will use the HTTP protocol, indicated by the "web" portion and our "java" section specifies it is a Java application. 
+The last portion of the Procfile creates our jar file. A jar file is a file used to run a java application. You can use a jar file to run your java code anywhere. It's easily exported and
+is how Heroku runs our application. 
+
+The next configuration we need a <strong> system.properties </strong> file. This is a file that Heroku uses to define our application's properties such as our Java and Maven version. 
+
+Our system.properties looks like
+```ignorelang
+java.runtime.version=11
+maven.version=3.6.2
+```
+Make sure this file is in our root directory so that Heroku can easily identify this file.
+
+Once we have all of our configurations set up, we can begin deploying our backend. Navigate to our backend directory and run the command:
+
+```shell
+mvn clean install
+```
+
+This will clean our application and build it again.
+Once we verified that our application successfully built, we follow the same steps as deploying our frontend code.
+
+We run the commands:
+
+```shell
+git add .
+```
+which adds all our files to our project's git repository
+
+```shell
+git commit -m
+```
+which commits our added files
```

Comment:
Don't forget to mention here that this should only be done if you just initialized a git repo or if they haven't committed these changes. Also, make them merge their changes to master